src/components/protected-route/protectedRoute.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Redirect, Route } from 'react-router-dom';
+
+const ProtectedRoute = ({
+  component: Component,
```

Comment:
Why do you need `component: Component`? Can you just have an argument with the name `Component`?

Reply:
you should pass the React Component as property, not sure if it would work if i made Component: Component